plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/AllManifestsTable.java

```diff
@@ -38,7 +39,7 @@
 public class AllManifestsTable
         extends BaseSystemTable
 {
-    public AllManifestsTable(SchemaTableName tableName, Table icebergTable)
+    public AllManifestsTable(SchemaTableName tableName, Table icebergTable, ExecutorService executorService)
```

Comment:
Please shorten `executorService` to `executor` everywhere

Reply:
Done.